.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor/
 /composer.lock
+/.idea
```

Comment:
I'm not sure we would like to add all editors support in our `.gitignore`. Is it possible for you to add it in your `.gitconfig`?